tools/deb/test/build_all.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# For e2e tests, build pilot, mixer, istio from head.
```

Comment:
Why should this be in proxy?

Reply:
That's all we're running in the VM so far - the proxy. 
Eventually we can move them to istio/istio - but I would rather do it after we sanitize
the build system, editing SHAs and uploading/downloading from gcr.io or docker is not the 
cleanest way.